app/models/moirai/translation_dumper.rb

```diff
@@ -0,0 +1,31 @@
+module Moirai
+  class TranslationDumper
+    def call
+      Moirai::Translation.pluck(:file_path).uniq.map do |file_path|
+        updated_file_contents = get_updated_file_contents(file_path)
+        {
+          file_path: file_path,
+          content: updated_file_contents
+        }
+      end
+    end
+
+    private
+
+    def get_updated_file_contents(file_path)
+      translations = Moirai::Translation.where(file_path: file_path)
```

Comment:
Idea: Introduce attr_accessor for Translation records to prevent N+1 queries, but this is not really necessary, as there is only 1 Query per file.